packages/nocodb/src/lib/meta/api/swagger/redocHtml.ts

```diff
@@ -5,7 +5,7 @@ export default `<!DOCTYPE html>
     <!-- needed for adaptive design -->
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
+    <link href="/css/fonts.montserrat.css" rel="stylesheet">
```

Comment:
what if nocodb is serving under a subpath (for eg:  using some proxy )? Since providing `/` in the beginning it will look under the root url.